Source/JavaScriptCore/API/glib/JSCValue.cpp

```diff
@@ -2145,3 +2145,30 @@ char* jsc_value_to_json(JSCValue* value, unsigned indent)
 
     return json;
 }
+
+/**
+ * jsc_value_new_deferred_promise:
+ * @context: a #JSCContext
+ * @value: a #JSCValue
```

Comment:
The list of parameters is missing `resolve` and `reject`, which need to be marked `(out) (transfer full) (nullable)` to indicate that they are new values returned by the function, that the caller takes their ownership, and that it is acceptable to pass `NULL`:

```suggestion
 * @value: a #JSCValue
 * @resolve: (out) (transfer full) (nullable): a callable #JSCValue.
 * @reject: (out) (transfer full) (nullable): a callable #JSCValue.
```